test/topology_random_failures/cluster_events.py

```diff
@@ -431,8 +433,9 @@ async def remove_data_dir_of_dead_node(manager: ManagerClient,
 
     yield
 
-    LOGGER.info("Remove data dir of the dead node and start it")
-    rmtree(path=data_dir, ignore_errors=True)
+    LOGGER.info("Remove data directories of the dead node and start it except conf dir")
+    for directory in data_dirs:
+        rmtree(path=directory, ignore_errors=True)
     await manager.server_start(server_id=running_servers[1].server_id)
```

Comment:
However, there is still a problem: this will try to bootstrap the node, but that will fail because there's a dead node in the cluster now.

There are two options out of this:
- remove the dead node first (using `removenode`)
- or here, perform a replace

Both options are valid, so we could have two cluster events even.

I would say let's go with the replace option.